sections/06_discussion.md

```diff
@@ -475,17 +475,12 @@ sharing and analysis [@doi:10.1038/ng.3830].
 The sharing of high-quality, labeled datasets will be especially valuable.  In
 addition, researchers who invest time to preprocess datasets to be suitable for
 deep learning can make the preprocessing code (e.g. Basset
-[@tag:Kelley2016_basset] and variationanalysis [@tag:Torracinta2016_deep_snp])
+[@tag:Kelley2016_basset] and variation analysis [@tag:Torracinta2016_deep_snp])
 and cleaned data (e.g. MoleculeNet [@tag:Wu2017_molecule_net]) publicly
 available to catalyze further research. However, there are complex privacy and
-legal issues involved in sharing patient data that cannot be ignored. In some
-domains high-quality training data has been generated privately, i.e.
-high-throughput chemical screening data at pharmaceutical companies. One
-perspective is that there is little expectation or incentive for this private
-data to be shared. However, data are not inherently valuable. Instead, the
-insights that we glean from them are where the value lies. Private companies may
-establish a competitive advantage by releasing data sufficient for improved
-methods to be developed.
+legal issues involved in sharing patient data that cannot be ignored. Solving
+these issues will require increased understanding of privacy risks and
+standards specifying acceptable levels. In some domains high-quality training data has been generated privately, i.e. high-throughput chemical screening data at pharmaceutical companies. One perspective is that there is little expectation or incentive for this private data to be shared. However, data are not inherently valuable. Instead, the insights that we glean from them are where the value lies. Private companies may establish a competitive advantage by releasing data sufficient for improved methods to be developed.
```

Comment:
suggesting seeing https://github.com/greenelab/deep-review/issues/576 and mentioning that via citation here, since you touched this anyway.